share/qtcreator/sfdk/modules/20-building-mb2.json

```
@@ -152,6 +152,21 @@
         {
             "name": "no-pull-build-requires",
             "trDescription": "Do not update build-time dependencies unless required explicitly with 'build-requires pull'."
+        },
+        {
+            "name": "package-signing-user",
```

```suggestion
            "name": "package.signing-user",
```

Similarly in the other two cases.